web/src/app/settings/setting-creator/setting-creator.component.ts

```diff
@@ -88,7 +88,23 @@ export class SettingCreatorComponent implements OnInit {
 
     validateSettingName() {
         this.setting.name = this.setting.name.trim();
-        this.isInputValid = !_.find(this.settings, s => s.name.trim() === this.setting.name);
+        this.isInputValid = this.isSettingNameUnique() && this.isSettingNameValid();
+    }
+
+    getValidationErrorMessage(): string {
+        if (!this.isSettingNameValid())
+            return 'Setting name should be more then 3 symbols';
```

Comment:
There is mistake in the word `then`.
Maybe we can find another massage:
`Setting name should be longer than ...` or 
`Setting name must contains at least ...`